tools/scalar-schema/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM clojure:openjdk-11-lein-slim-buster AS BUILD
+COPY . /code
+WORKDIR /code
+RUN lein uberjar
+
+FROM openjdk:11-jre-slim
```

Comment:
Please use openjdk 8. (maybe 8u302?)

Reply:
I have updated to code and for now just switched to the same one used as in the scalardl schema loader.